### DISCUSSION TITLE
Localisation

### DISCUSSION THREAD

**Discussion by mafineeek** [stoatchat/discussions#303] [Feature Suggestions]
Hii!! I think adding user current locale to user object will be nice idea and will help in localisation (it will be easier)

**Comment by AllanGibbs** [comment]
Speakting of localisation ... 
Perhaps it is possible to use Crowdin to create a natural native sounding GUI. Properly translated GUI elements, no AI-nonsense. I have a very good experience with Crowdin.

**Reply by JLUsr** [reply]
> Perhaps it is possible to use Crowdin...

Stoat does have a [webslate](https://translate.stoat.chat/engage/revolt/) page you can use. I don't know if this is exactly what you are after but it seems comparable. Going to Stoats GitHub Overview page you can find [Developer Documentation](https://developers.stoat.chat/), this was the only way I could find this when I looked so it is pretty easy to overlook.
